scripts/check_all_metrics.py

```diff
@@ -0,0 +1,122 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import argparse
+import logging
+import os
+import subprocess
+import sys
+from fnmatch import fnmatch
+from pathlib import Path
+from typing import List
+
+import taskcluster
+
+from bugbug.utils import get_taskcluster_options
+
+LOGGER = logging.getLogger(__name__)
+
+logging.basicConfig(level=logging.INFO)
+
+QUEUE_ROUTE_PATTERN = "index.project.relman.bugbug.train_*.per_date.*"
+
+CURRENT_DIR = Path(__file__).resolve().parent
+
+
+def download_metric(model_name: str, metric_directory: str):
+    download_script_path = CURRENT_DIR / "retrieve_training_metrics.py"
+
+    cli_args = [
+        sys.executable,
+        str(download_script_path),
+        model_name,
+        "2019",
+        "-d",
+        metric_directory,
+    ]  # type: List[str]
+
+    LOGGER.info("Download metrics for %r", model_name)
+
+    subprocess.check_output(cli_args)
```

Comment:
Nit: subprocess.run(cli_args, check=True)

Reply:
Do you prefer not to see the process output?

Reply:
`subprocess.run(cli_args, check=True)` shows the output, `subprocess.check_output(cli_args)` doesn't.

Reply:
Got it, it is catching stdout but not stderr and most output how the other scripts are on stderr, I will make the change.